src/test/java/com/hazelcast/kubernetes/DefaultKubernetesClientTest.java

```diff
@@ -62,7 +63,7 @@
     @Before
     public void setUp() {
         String kubernetesMasterUrl = String.format("http://%s:%d", KUBERNETES_MASTER_IP, wireMockRule.port());
-        kubernetesClient = new DefaultKubernetesClient(kubernetesMasterUrl, TOKEN);
+        kubernetesClient = new DefaultKubernetesClient(kubernetesMasterUrl, TOKEN, CA_CERTIFICATE);
```

Comment:
should we have separate test cases for `CA_CERTIFICATE == null` vs a string?

Reply:
Good idea. Added.